pkg/frontend/mysql_cmd_executor.go

```diff
@@ -1551,9 +1551,23 @@ func (cwft *TxnComputationWrapper) GetAffectedRows() uint64 {
 
 func (cwft *TxnComputationWrapper) Compile(u interface{}, fill func(interface{}, *batch.Batch) error) (interface{}, error) {
 	var err error
-	cwft.plan, err = plan2.BuildPlan(cwft.ses.GetTxnCompilerContext(), cwft.stmt)
-	if err != nil {
-		return nil, err
+	switch stmt := cwft.stmt.(type) {
```

Comment:
ParenSelect Optimiser also required

Reply:
I will add it.

Reply:
Delete, Update, Insert also required to call optimizer

Reply:
I have added it.